package.json

```diff
@@ -38,6 +38,8 @@
     "get-stdin": "^6.0.0"
   },
   "devDependencies": {
+    "@typescript-eslint/eslint-plugin": "1.0.0",
+    "@typescript-eslint/parser": "^1.0.0",
```

Comment:
The versions under @typescript-eslint should match each other exactly

Reply:
I got it. I fixed!